src/main/resources/assets/robotics/lang/no_no.json

```
@@ -0,0 +1,75 @@
+  "_": "->------------------------]  Spill Elementer  [------------------------<-",
+
+  "block.robotics.bronze_block": "Bronseblokk",
+  "block.robotics.code_editor": "Koderedigerer",
+  "block.robotics.deepslate_tin_ore": "Tinnmalm i dypskifer",
+  "block.robotics.smasher": "Knuser",
+  "block.robotics.tin_block": "Tinnblokk",
+  "block.robotics.tin_ore": "Tinnmalm",
+
+  "entity.robotics.clockcopter": "Klokkekopter",
+  "entity.robotics.code_drone": "Kodedrone",
+  "entity.robotics.extend_o_boots": "Ekstend O Støvler",
+  "entity.robotics.miner": "Gruvearbeider",
+
+  "item.robotics.bronze_ingot": "Bronsebarre",
+  "item.robotics.bronze_nugget": "Bronseklump",
+  "item.robotics.clockcopter": "Klokkekopter",
+  "item.robotics.code_drone": "Kodedrone",
```

The Name of the Code Drone is Juan, I'd suggest leaving it as-is:
```suggestion
  "item.robotics.code_drone": "Juan",
```